### PR TITLE
Fixes #18593: Add ubuntu18 box to the packer samples

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/18593

### CODE REVIEW

File: packer/systems/ubuntu/scripts/os_start.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -xe
+
+export DEBIAN_FRONTEND=noninteractive
+# debian 6 keys are now too old
+grep 6.0 /etc/debian_version && EXTRA_OPT="--allow-unauthenticated"
```
Comment (peckpeck) [inline packer/systems/ubuntu/scripts/os_start.sh:7]: debian specific on ubuntu ?

**Comment by Fdall** [comment]
Commit modified

File: packer/systems/ubuntu/scripts/os_start.sh (hunk 1 above)
Comment (Fdall) [reply packer/systems/ubuntu/scripts/os_start.sh:7]: bad copy/paste

**Comment by Fdall** [comment]
PR updated with a new commit

**Review by peckpeck — APPROVED** [review]

**Pull request merged**